demo/responsive.html

```diff
@@ -1,74 +0,0 @@
-<!doctype html>
```

Comment:
Deleted a bunch of the other demos as they were redundant with what we have already and were created for the old Galen tests.

Reply:
Yeah not super concerned with full coverage here if we have it in `core` and the idea is to slowly remove these usages anyways